pyzeebe/grpc/__init__.py

```diff
@@ -0,0 +1,13 @@
+GRPC_CHANNEL_OPTIONS = {
```

Comment:
Sure. Reason I put this here was that I wanted to allow for monkeypatching, without touching "internals". `grpc` sounds more like a public API. 

But having an option to set up the gRPC channel outside of Pyzeebe and passing it should probably be the official way, so I agree that it can be moved.